ports/esp32/README.ulp.md

```diff
@@ -119,10 +119,8 @@ import machine
 import time
 
 u = machine.ULP()
-f = open('test.bin', 'rb')
-b = f.read()
-
-u.load_binary(0,b)
-
-u.run(0)
+with open('test.bin', 'rb') as f:
+	b = f.read()
+	u.load_binary(0,b)
+	u.run(0)
```

Comment:
are these TABs instead of 4 spaces?

Reply:
also, maybe outdent the load and run, so it is not within the context manager (you already have what you need in b, so the file can be already closed before load/run).